.github/workflows/publish_releasenotes_outside.yml

```diff
@@ -0,0 +1,26 @@
+#############################################################################
+# GitHub Action to Publish Release Notes GitHub Page.
+#
+#############################################################################
+name: "Publish Release notes GitHub Page"
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'Target Version without (-SNAPSHOT)'
+        required: true
+jobs:
+  publish:
+    name: Publish releasenotes GitHub Page ${{ github.event.inputs.version }}
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout the latest code
+        uses: actions/checkout@v2
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+      - name: Download xmlstarlet
+        run: |
+          sudo apt-get install xmlstarlet
```

Comment:
It is not required

Reply:
To get the current release, it is required. Please check script